examples/typescript-react/js/app.tsx

```diff
@@ -0,0 +1,175 @@
+/*jshint quotmark:false */
+/*jshint white:false */
+/*jshint trailing:false */
+/*jshint newcap:false */
+/*global React, Router*/
+
+/// <reference path="../typings/tsd.d.ts" />
+declare var Router : any;
+
+var TodoModel = app.models.TodoModel;
+var TodoFooter = app.components.TodoFooter;
+var TodoItem = app.components.TodoItem;
+
+namespace app.components {
+
+  export class TodoApp extends React.Component<IAppProps, IAppState> {
+
+    constructor(props : IAppProps) {
+      super(props);
+      this.state = {
```

Comment:
Is `state` ever declared? Even if you still get output in the face of errors, I'd suggest adding a property declaration at the top.